src/main/java/org/opensrp/repository/postgres/mapper/custom/CustomEventMapper.java

```diff
@@ -27,7 +27,4 @@
 	
 	List<Event> selectByConceptAndValue(@Param("concept") String concept, @Param("conceptValue") String conceptValue);
 
-	List<Event> selectByBaseEntityIdAndDetails(@Param("baseEntityId") String baseEntityId, @Param("detailKey") String detailKey,
```

Comment:
You can leave this method as it can useful in the future

Reply:
Sure